### PR TITLE
docs(readme): update the readme file and remove extra contents

### PR DESCRIPTION
**Pull request by parasss19**
## Closes: #481

- [ x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ x] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update.
- [ ] This change requires a documentation update

## Checklist

- [x ] My code follows the guidelines of this project.
- [x ] I have performed a self-review of my own code.
- [x ] I have commented on my code, particularly wherever it was hard to understand.
- [ x] I have made corresponding changes to the documentation.
- [ x] My changes generate no new warnings.
- [x ] I have added tests that prove my fix is effective or that my feature works.
- [x ] Any dependent changes have been merged and published in downstream modules.

## Code of Conduct

- [x ] I agree to follow this project's [Code of Conduct](https://github.com/amanjaiman1/Product_3D/blob/main/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,72 +1,137 @@
...
+git clone https://github.com/amanjaiman1/Product_3D.git
+```
+
+This command will create a local copy of the project on your machine.
+
+2.  Navigate to the project folder
+
+```
+cd ProjectsHut
+```
```
Comment (balaji-sivasakthi) [inline CONTRIBUTING.md:21]: Kindly, add the step for the fork.

File: README.md [hunk 2]
```diff
@@ -1,30 +1,34 @@
...
-  <img src="https://img.shields.io/github/issues-pr/amanjaiman1/Product_3D?style=for-the-badge" /><br>
-  <img src="https://img.shields.io/github/issues-pr-closed-raw/amanjaiman1/Product_3D?style=for-the-badge" />
-  <img src="https://img.shields.io/github/stars/amanjaiman1/Product_3d?style=for-the-badge" />
-  <img src="https://img.shields.io/github/contributors/amanjaiman1/Product_3D?style=for-the-badge" />
-  <img src="https://img.shields.io/github/last-commit/amanjaiman1/Product_3D?style=for-the-badge" />
-  </div>
-
-# 3D Product Customization Website 🎨
+ <div align="center">
+ <img src="https://github.com/amanjaiman1/Product_3D/blob/main/public/icons8-shirt-58.png?raw=true" alt="logo"/>
```
Comment (balaji-sivasakthi) [inline README.md:2]: Don't use the GitHub link, It might not work on others.

**Comment by parasss19** [comment]
Hey @balaji-sivasakthi 
- Added the logo img
- Added fork step
- Removed DalleAi text
- Added issues Guidelines

**Comment by balaji-sivasakthi** [comment]
@amanjaiman1 merge it!

**Review by amanjaiman1 — APPROVED** [review]

**Pull request merged**